pkg/cluster/cluster.go

```diff
@@ -645,6 +653,10 @@ func (c *Cluster) initHumanUsers() error {
 			}
 		}
 
+		if _, present := c.pgUsers[username]; present {
+			c.logger.Warnf("overwriting existing user %q with the data from the teams API")
```

Comment:
1. Should the username be passed to the Warnf call as a value ?
2. Is the data from the teams API the only thing that can trigger the overwrite ?

Reply:
good catch for the username, teams API roles will overwrite roles with the same name obtained from the infrastructure roles or the cluster manifests, but cluster manifest roles will merge with the infrastructure one (same PR), so teams API roles are the only ones that can overwrite other roles.